cellprofiler/modules/enhanceorsuppressfeatures.py

```diff
@@ -556,6 +556,9 @@ def enhance_dic(self, image, angle, decay, smoothing):
 
             return result
 
+        if smoothing == 0:
```

Comment:
I found that the issue was bubbling up from inside centrosome (i.e. centrosome wasn’t protecting the denominator) so I added this to address the problem. It has the benefit of not modifying the existing `smoothing` behavior.